core/src/com/po/fuck/model/collision/Collidable.java

```diff
@@ -11,6 +11,8 @@ public interface Collidable {
 
     Polygon getCollision();
 
+    boolean isInvisible();
```

Comment:
What does it mean ? Very strange logic, it does not even affect `collide`.

Reply:
It affects it. `return objects.stream().filter(x -> !x.isInvisible() && x.collide(other)).collect(Collectors.toList());`

Reply:
But what does `invisible` state mean ? If it still collides, and it only affects physics.